packages/client/src/store/modules/grants.js

```diff
@@ -48,6 +95,30 @@ export default {
       return fetchApi.get(`/api/organizations/:organizationId/grants?${query}`)
         .then((data) => commit('SET_GRANTS', data));
     },
+    fetchGrantsNext({ commit }, {
```

Comment:
Using a new function name here so we don't affect the existing Grants Table functionality.